container-images/llama-stack/Containerfile

```diff
@@ -9,9 +9,9 @@ RUN dnf -y update && \
     dnf -y install uv cmake gcc gcc-c++ python3-devel pkg-config sentencepiece-devel && \
     dnf -y clean all
 
-RUN uv pip install --system ramalama-stack
-
-RUN uv run llama stack run ~/.llama/distributions/ramalama/ramalama-run.yaml --image-type venv --image-name /venv
+RUN uv venv  && \
```

Comment:
Could be interesting to do a plain-old pip install here.

In terms of uv, I'm kinda liking this command recently `uv tool install`:

uv tool install --force --python python3.12 ramalama

if the python package in an executable in puts the tool in your PATH somewhere so you don't have to prefix "uv run".

Reply:
Seems like llama-stack is insisting on being run in a /.venv type environment.

Reply:
Yeah not sure how we do it for this case I think it's because ramalama-stack is just a library and llama-stack is a dependancy, if we install llama-stack as a "tool" it works.